stock_picking_partner_invoice_address/views/stock_picking_view.xml

```diff
@@ -0,0 +1,44 @@
+<odoo>
+    <record id="view_picking_form" model="ir.ui.view">
+        <field name="name">stock.picking.form</field>
+        <field name="model">stock.picking</field>
+        <field name="inherit_id" ref="stock.view_picking_form" />
+        <field name="arch" type="xml">
+            <xpath expr="//field[@name='partner_id']" position="after">
+                <field name="partner_invoice_id"/>
+                <field name="same_invoiced_id"/>
```

Comment:
Show these fields only for outgoing pickings.

Reply:
I fixed this.